packages/web-runtime/src/container/sse/files.ts

```diff
@@ -153,13 +153,14 @@ export const onSSEItemTrashedEvent = ({
     // If initiated by current client (browser tab), action unnecessary. Web manages its own logic, return early.
     return
   }
+  const { $gettext } = language
```

Comment:
if this works, you can use this approach above where you need to access `language.current`